llvm/include/llvm/CodeGen/TargetLowering.h

```diff
@@ -3155,8 +3155,9 @@ class TargetLoweringBase {
   ///
   /// \p DI is the deinterleave intrinsic.
   /// \p LI is the accompanying load instruction
```

Comment:
The comments on these 2 functions look outdated. It would be nice to explain what DeadInsts should look like going into and out of the function given it's a reference to a vector.

Reply:
Thanks!